feat(docs): clean up app transfer docs (#18968)
expo/expo: docs/pages/distribution/app-transfers.md

---
title: App Transfers
hideTOC: true
---

import { BoxLink } from '~/ui/components/BoxLink';

> There are two different representations of your app to consider when handing over ownership to another entity: the app as it exists on Expo Application Services (in order to create builds with EAS Build, send updates with EAS Update, etc.) and the app records on the Apple App Store / Google Play Store (in order to distribute the app to end-users). The following guides explain how to handle app transfers in each of these cases.

Reviewer: ```suggestion
There are two different representations of your app to consider when handing over ownership to another entity: the app as it exists on Expo Application Services (to create builds with EAS Build, send updates with EAS Update, etc.) and the app records on the Apple App Store/Google Play Store (to distribute the app to end-users). The following guides explain how to handle app transfers in each case.
```

As per our style guide, let's [avoid adding spaces before and after `/`](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#do-not-add-a-space-between-the-words-and-).

---
title: App Transfers
hideTOC: true
---

import { BoxLink } from '~/ui/components/BoxLink';

> Universal apps using EAS have two parts:

Reviewer 2: ```suggestion
There are two different representations of your app to consider when handing over ownership to another entity: the app as it exists on Expo Application Services (in order to create builds with EAS Build, send updates with EAS Update, etc.) and the app records on the Apple App Store / Google Play Store (in order to distribute the app to end-users). The following guides explain how to handle app transfers in each of these cases.
```

> 1. JavaScript files and assets which are hosted with [EAS Update](/eas-update/introduction). These static files are used for over-the-air updates.
> 2. [Native runtime](/workflow/glossary-of-terms#native-runtime) hosted in the respective app stores. Used for rendering the static hosted assets.

Reviewer 2: ```suggestion
```

> Learn more in the official guides for each:

Reviewer 2: ```suggestion
```
